packages/cinterop/src/androidTest/java/io/realm/interop/CinteropTest.kt

```diff
@@ -236,4 +239,117 @@ class CinteropTest {
 
         realmc.realm_commit(realm)
     }
+
+    @Test
+    fun parentChildRelationship() {
+        val context = InstrumentationRegistry.getInstrumentation().context
+
+        System.loadLibrary("realmc")
+        println(realmc.realm_get_library_version())
```

Comment:
The `CinteropTest` was primarily used for ramping up the interop layer towards the C-API, so not sure they should stay, but I have found it useful to pin point issues in the C-API without any generated schema/accessors, so maybe it could stay in some form. Anyway, I extracted the tests and verified the output instead of printing.